### ISSUE TITLE
[메인펑션리스트 개발팀 회의] 1. 욕봇 (레퍼런스 파일 첨부했습니다. 꼭 읽어봐주세요)

### ISSUE THREAD

**Issue by saeromCho** [drfts/drfts#7]
1. 욕봇 

 레퍼런스를 보시면 아시겠지만 형태소별로 분석해야 한다는 점(시발 과 ㅅ1bal 두가지 경우 전부 변환이 필요합니다.), 문장의 의미가 무엇인지에 따라 해당 단어의 변환여부가 달라진다는 점 등 다양한 경우의 수를 고려해야 합니다. 
저번주 4월 8일 개발모임에서 용근님, 진희님과 욕봇에 대해 잠깐 다루었습니다. 머신러닝을 이용할 예정이며, 초반엔 데이터를 집어 넣어놓고, 추후엔 점차 들어오는 데이터들을 기반으로 점점 학습을 시킨다는 정도로만 얘기가 되었습니다. 앞으로 욕봇 구현 진행방향에 대해 얘기하고 싶습니다.
지연님이 주신 레퍼런스 파일 참고 바랍니다.
[욕봇 레퍼런스.pdf](https://github.com/drfts/drfts/files/1895551/default.pdf)
[서강대학교 머신러닝 최종보고서.pdf](https://github.com/drfts/drfts/files/1895553/default.pdf)

**Comment by jinheebab** [comment]
순서를 정합시다. 뭐부터 건드려볼지. 그때 얘기해요.

**Comment by insaint03** [comment]
형태소 분석부터 시작하면 일이 좀 커질 것 같은데;;

초기에는 키워드 중심으로 대응하고, 당분간은 https://aws.amazon.com/ko/lex/ 나 https://www.ibm.com/watson/ 같은 상용 플랫폼을 활용하는 방안을 생각해 보죠. 물론 장기 계획으로 학습 모델을 자체 구축할 수는 있겠지만 지금 세부 내용을 이야기하기에는 시기상조로 보입니다.

**Comment by saeromCho** [comment]
네 맨 아래에 적어놨지만 욕봇에 대한 진행방향을 다같이 얘기해보자는 의도였습니다. 기획팀분들께는 개발상으로 가능한지 불가능한지를 우선 전달드려야하는 부분입니다. 얘기도 들어봤고, 어느정도 얘기가 오갔으니 욕봇은 가능한 부분이라고 전달드리겠습니다.

**Comment by jb-** [comment]
욕 여부의 판단(1)과 그것의 치환(2)은 전혀 별개의 문제입니다. 
Drfts가 유저에게 제공하고자 하는 경험에서 (2)는 어느 정도의 중요도를 갖는지요?

(2)가 중요하다면 (1)이 허접한 상태일지언정 어떻게든 (2)를 구현해야겠죠.
(2)가 중요하지 않다면, (1)에 우선 집중하는 게 맞을 거고요.

**Comment by jinheebab** [comment]
유저에게 제공해야하는 경험과
서비스 자체적인 성격으로 보았을때 1이 더 중요합니다.
